spec/acceptance/users_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'acceptance_helper'
+
+feature 'Show user profiles', type: :feature, js: true do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.